src/client/components/Home/RegisterButton.style.css

```diff
@@ -0,0 +1,18 @@
+.homepage-slogan{
+  height: 63vh;
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
+
+.homepage-slogan:nth-of-type(1){
```

Comment:
Wouldn't be better to consider the 63% opacity as well?

Reply:
if i give the opacity 63 it's very light and it's not looking good as in figma.
![home1](https://user-images.githubusercontent.com/66561373/107624325-fc426c00-6c5a-11eb-9204-d9ec5f1122b3.png)

Reply:
added the opacity of 75%